pkgs/build-support/node/build-npm-package/hooks/default.nix

```
@@ -27,7 +27,10 @@
   npmInstallHook = makeSetupHook
     {
       name = "npm-install-hook";
-      propagatedBuildInputs = [ buildPackages.makeWrapper ];
+      propagatedBuildInputs = [
+        buildPackages.installShellFiles
+        buildPackages.makeWrapper
+      ];
```

I think this looks nicer:
```suggestion
      propagatedBuildInputs = with buildPackages; [
        installShellFiles
        makeWrapper
      ];
```